### DISCUSSION TITLE
rp2040 pico w-2023

### DISCUSSION THREAD

**Discussion by kipigor7** [fhdm-dev/scoppy#143] [General]
I bought an rp2040 pico w-2023 board(https://www.aliexpress.com/item/1005006125974166.html ). But as it turns out, it costs es8285. It’s not clear whether it’s possible to make firmware for Scoppy it?

**Comment by fhdm-dev** [comment]
Hi,
I don't have one of those boards but I think it's very unlikely it will work with the wireless version of the Scoppy firmware :(

**Comment by vvc1950** [comment]
Hi! This is possible if you change the firmware in terms of sharing over WiFi. ;-D

**Comment by kipigor7** [comment]
It would be great, but unfortunately I'm not good at programming:(
